.github/workflows/status-comment.yml

```
@@ -0,0 +1,80 @@
+            message="❓ The $WORKFLOW_NAME workflow has completed with an unknown status."
+          fi
+          echo "message=$message Check the [workflow run]($WORKFLOW_URL) for details. ($SHA)" >> "${GITHUB_OUTPUT}"
+      # Finally, post the status comment on the PR
+      - name: Update CI Status
+        uses: marocchino/sticky-pull-request-comment@v2
+        if: ${{ github.event.workflow_run.event == 'pull_request' }}
+        with:
+          header: ${{ github.event.workflow_run.name }}
+          hide_details: true
+          number: ${{ steps.pr-context.outputs.number }}
+          message: ${{ steps.set-message.outputs.message }}
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)